mail_client/api/mail.py

```diff
@@ -88,13 +90,11 @@ def get_incoming_mails(start: int = 0) -> list:
 def get_outgoing_mails(start: int = 0) -> list:
 	"""Returns outgoing mails for the current user."""
 
+	mailboxes = frappe.get_all("Mailbox", {"user": frappe.session.user}, pluck="name")
```

Comment:
You can use utils.user.get_user_mailboxes() to get the user's mailboxes from the cache. One case is that mailbox has enable incoming and outgoing thing.